src/viperleed/calc/lib/fortran_utils.py

```diff
@@ -11,10 +11,18 @@
 __created__ = '2024-08-26'
 __license__ = 'GPLv3+'
 
+import subprocess
+
+from .version import Version
 
 _FORTRAN_LINE_LENGTH = 72  # FORTRAN line-length limit
 _F77_CONTINUATION_POS = 6  # Column of the continuation character
 
+class MpifortNotFoundError(Exception):
```

Comment:
I would perhaps make these exceptions not related to `mpifort`, since they may be used later on for other compilers. I'd suggest: `CompilerNotFoundError` and `NoCompilerVersionFoundError`. I would make them both a subclass of a more general `FortranCompilerError` exception.